ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-bgp-evpn-mpls.md

```diff
@@ -122,6 +122,7 @@ router bgp 65101
    graceful-restart restart-time 300
    graceful-restart
    maximum-paths 2 ecmp 2
+   neighbor default send-community standard large extended
```

Comment:
`
s1-leaf1(config-router-bgp)#neighbor default send-community large ?
  <cr>  

s1-leaf1(config-router-bgp)#neighbor default send-community large extended
% Invalid input
`

Combination is not working for me.

Reply:
yes there is some restriction of using the community, if we have given large at the start then cant give other, plus if extended as first then only large can be added and  so on. 
So @Shivani-gslab just update the input

Reply:
Fixed